### PR TITLE
Merging asr_streaming.sh into asr.sh 

### PR DESCRIPTION
**Pull request by D-Keqi**
Merging asr_streaming.sh into asr.sh.

### CODE REVIEW

**Comment by D-Keqi** [comment]
Hi! @ftshijt  I make a new PR to merge the asr_streaming.sh into the asr.sh. 
The old one https://github.com/espnet/espnet/pull/3806 has some conflicts.
After this PR is merged, I think we can easily merge that PR https://github.com/espnet/espnet/pull/3806.

File: egs2/TEMPLATE/asr1/asr.sh [hunk 1]
```diff
@@ -108,6 +108,8 @@ nll_batch_size=100 # Affect GPU memory usage when computing nll
                    # during nbest rescoring
 k2_config=./conf/decode_asr_transformer_with_k2.yaml
 
+use_streaming=false # Whether to use streaming decoding
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/asr.sh:111]: Looks great, could you also update the help_message

Comment (ftshijt) [reply egs2/TEMPLATE/asr1/asr.sh:111]: After you change this, I will merge the PR

Comment (D-Keqi) [reply egs2/TEMPLATE/asr1/asr.sh:111]: > 

Sorry, I'm not sure what you mean by "help_message". Does it mean my comment "# Whether to use streaming decoding"? How should I modify it?

Comment (D-Keqi) [reply egs2/TEMPLATE/asr1/asr.sh:111]: > 

Sorry, I'm not sure what you mean by "help_message". Does it mean my comment "# Whether to use streaming decoding"? How should I modify it? @ftshijt

Comment (ftshijt) [reply egs2/TEMPLATE/asr1/asr.sh:111]: Just update the string defined here https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L144 would be fine.

Comment (D-Keqi) [reply egs2/TEMPLATE/asr1/asr.sh:111]: > 

I see! thanks, I have updated it.

**Comment by ftshijt** [comment]
Very cool! Many thanks

**Pull request merged**